.github/workflows/macos-llvm-clang.yaml

```diff
@@ -23,6 +23,6 @@ jobs:
         uses: ./.github/actions/setup-spack-stack
         with:
           app: ${{ matrix.app }}
-          compiler: apple-clang
+          compiler: clang
```

Comment:
Note to self. Do we need to add to PATH and LD_LIBRARY_PATH to find homebrew's llvm-clang?

Reply:
Will figure out when trying the CI tests.